src/components/Graph.js

```diff
@@ -10,38 +10,36 @@ const Graph = (props) => {
   let percentComplete = length > 0 ? countComplete/count*100 : 0
   let percentTodo = length > 0 ? countTodo/count*100 : 0
   let percentInProgress = length > 0 ? countInProgress/count*100 : 0
+  let graphData = [ { name:'Complete',
+                      count:countComplete,
+                      percent:percentComplete,
+                      color:'#3EC556' },
+                    { name:'In Progress',
+                      count:countInProgress,
+                      percent:percentInProgress,
+                      color:'#3E74C5' },
+                    { name:'Todo',
+                      count:countTodo,
+                      percent:percentTodo,
+                      color:'#C53E3E' } ]
+                      
+  let graphList = graphData.map((graph) => {
+                    return (
+                      <div key={ graph.name + graph.color } className='col-md-4 graph-border'>
+                        <div className='text-center'>
+                          <p className='center-text-horizontal graphOne graphNumber' > { graph.count } </p>
+                          <p className='center-text-horizontal graphText' > { graph.name } </p>
+                        </div>
+                        <p className='percent'>{ Math.floor(graph.percent) }%</p>
+                        <div className='col-md-6 col-centered'>
+                          <Circle percent={ graph.percent } strokeWidth='10' strokeColor={ graph.color } />
+                        </div>
+                      </div>
+                    )
+                  })             
```

Comment:
This is one way to reduce repetition, it definitely works and cleans things up a bit.
Also, a component that doesn't need state or interaction can be made as a pure function that takes props and returns a dom element. You almost have that here so things to consider would be to actually make a function like that and map like you do here with the result being a single `Graph`. So if this component was renamed `GraphList` then you could call the function I'm describing `Graph` and have the map just build up a list of `<Graph>`.